mailu/templates/pvc.yaml

```diff
@@ -1,15 +1,16 @@
 # This file is derived from https://github.com/Mailu/Mailu/blob/master/docs/kubernetes/mailu/pvc.yaml
 
+{{- if ( eq "hostPath" (required "persistence.type" .Values.persistence.type)) }}
+# region PVC
+# create new PVC for hostPath persistence.type
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: {{ include "mailu.fullname" . }}-storage
```

Comment:
@micw I'm reverting this too, according to your other feedback (to be consistent)